### PR TITLE
Update results to weigh by citation counts

### PR DESCRIPTION
**Pull request by trangdata**

### CODE REVIEW

**Review by cgreene — COMMENTED** [review]
Where did the citation counts come from? That probably needs to be in the methods.

Also, I think we should note that we're using citation counts in the intro & give rationale (work may not be equally impactful) + that analyzing without citation counts leads to even more pronounced differences (+ a defense of why citation counts may be biased in the same way as honors and so reality is likely to be worse than we present here).

**Comment by trangdata** [comment]
I added a paragraph of explanation [here](https://github.com/greenelab/iscb-diversity-manuscript/pull/123/commits/c751b5f4a59833ad6229181191c8e8ed340b41ad). Maybe @cgreene or @dhimmel can revise!

File: content/30.methods.md [hunk 1]
```diff
@@ -35,8 +35,14 @@ PubMed is a search engine resource provided by the US National Library of Medici
...
+From PubMed, we compiled a catalog of 176,773 journal articles that were published from 1993 through 2019, written in English, and tagged with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+), which is [equivalent](https://www.ncbi.nlm.nih.gov/mesh/68019295) to "bioinformatics" and includes categories such as genomics and systems biology (via PubMed's term explosion to include subterms).
+Excluding 663 articles with no author information and years with less than 200 articles/year, we analyzed 176,110 articles from 1998--2019.
+
+We extracted the number of times an article has been cited by PubMed Central articles from the `PmcRefCount` of the PubMed DocSum XML records.
+Under the rationale that authors of more impactful articles have higher chance of being recognized, we used the PubMed Central citation count as a relative estimate of an article's impact to weigh each author's prediction probability (please see the Statistical Analysis subsection).
+However, we note the potential bias that comes with this citation count weighing approach,
```
Comment (cgreene) [inline content/30.methods.md]:
```suggestion
However, we note a potential bias for citation count weighing:
```

File: content/30.methods.md [hunk 2]
```diff
@@ -35,8 +35,14 @@ PubMed is a search engine resource provided by the US National Library of Medici
...
+such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].
```
Comment (cgreene) [inline content/30.methods.md]:
```suggestion
female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].
To the extent that there may be a bias in citation practices, we should expect that the bias for honorees would align with one for citations, meaning that our analysis would understate the severity of disparities.
```

**Review by cgreene — APPROVED** [review]
Looks good!

**Comment by cgreene** [comment]
I think that we are going to want to make one small change in that we'll want to move the interpretation part of the new text to the conclusions - likely the caveats paragraph. It seems easier to merge this now and do the move in a subsequent PR.

**Comment by cgreene** [comment]
Well - let me give it a shot - 1 sec

**Comment by cgreene** [comment]
Ok - see what you think with these changes!

File: content/30.methods.md [hunk 3]
```diff
@@ -38,6 +38,12 @@ To automate and generalize these tasks, we created the [pubmedpy](https://github
...
 Excluding 663 articles with no author information and years with less than 200 articles/year, we analyzed 176,110 articles from 1998--2019.
 
+We extracted the number of times an article has been cited by PubMed Central articles from the `PmcRefCount` of the PubMed DocSum XML records.
+Under the rationale that authors of more impactful articles have higher chance of being recognized, we used the PubMed Central citation count as a relative estimate of an article's impact to weigh each author's prediction probability (please see the Statistical Analysis subsection).
+However, we note the potential bias that comes with this citation count weighing approach,
+such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].
```
Comment (dhimmel) [inline content/30.methods.md]:
```suggestion
such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
```

File: content/25.discussion.md [hunk 4]
```diff
@@ -16,6 +16,11 @@ We acknowledged that our supervised learning approaches are neither error free n
 Because wru was trained on the US census to make predictions, many of the missing predictions are on names not observed in the US census.
 Although the underestimation of the proportion of these names could not be quantified in the list of authors and honorees, we complemented this race/ethnicity prediction method with an additional name origin analysis.
 By integrating different methods and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we hope to alleviate these biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
+We also weighted work using citation count to control for the differential impact of research contributions.
+However, we note a potential bias for using citation counts: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].
```
Comment (dhimmel) [inline content/25.discussion.md]:
```suggestion
However, we note a potential bias for using citation counts: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
```

**Review by dhimmel — COMMENTED** [review]
nice diff! Manubot working well here to show changes!

**Comment by trangdata** [comment]
Looks great!

**Pull request merged**